opts/ip.go

```diff
@@ -27,7 +29,7 @@ func (o *IpOpt) Set(val string) error {
 	return nil
 }
 
-func (o *IpOpt) String() string {
+func (o *IPOpt) String() string {
```

Comment:
This will also need godoc (stdlib documents `String()` methods as well, ref https://golang.org/src/bytes/buffer.go)

"String returns the IP address stored in the IPOpt. If IPOpt is a nil pointer, it returns an empty string."